### PR TITLE
more rtc features

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/i2s.rs
Comment (therealprof) [inline src/i2s.rs:1]: This change looks unrelated?

Comment (burrbull) [reply src/i2s.rs:1]: clippy/warning fix for previous PR

**Review by therealprof — APPROVED** [review]
LGTM, thanks.

bors r+

**Comment by burrbull** [comment]
bors retry
